zstacklib/zstacklib/test/utils/misc.py

```
@@ -12,14 +11,30 @@ def uuid():
     return str(uid.uuid4()).replace('-', '')
 
 
+def make_context_dict(api_msg_name, api_id=uuid(), timeout=1800):
```

Ruff (0.11.9)

Do not perform function call `uuid` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)